Formula/dart-sdk.rb

```diff
@@ -0,0 +1,39 @@
+class DartSdk < Formula
+  desc "Dart Language SDK, including the VM, dart2js, core libraries, and more"
+  homepage "https://dart.dev"
+  url "https://github.com/dart-lang/sdk/archive/refs/tags/2.17.3.tar.gz"
+  sha256 "10fdcb6980aee7d5d04b278f337f3519af7fa6bfa6290d737ea9327954b515a1"
+  license "BSD-3-Clause"
+
+  depends_on "ninja" => :build
+  depends_on "rust" => :build
+
+  uses_from_macos "curl" => :build
+  uses_from_macos "python" => :build
+
+  resource "depot-tools" do
+    url "https://chromium.googlesource.com/chromium/tools/depot_tools.git",
+        revision: "138bff2823590b3f3db440425bf712392defb7de"
+  end
```

Comment:
is this always using the latest revision of the `depot_tools`?

Reply:
I guess yes. They don't seem to have tag, so better than nothing.
I am wondering if we really need to ship that dependency ...

Reply:
AFAIK we don't ship the tools themselves. It's just needed to build.